src/Haipa.Modules.Identity/IdentityModule.cs

```diff
@@ -52,29 +117,69 @@ protected override void ConfigureServices(IServiceProvider serviceProvider, ISer
             //        options.RequireHttpsMetadata = false;
             //    });
 
-            services.AddAuthorization(options =>
```

Comment:
With these changes there is currently no auth in identity service. But authentication will also be required for management of clients.

Reply:
please let us speak the authorization in detail